### PR TITLE
Combine driver's .travis stages with operator's .travis stages

### PR DESCRIPTION
**Pull request by dunnevan**
takes old IBM/ibm-spectrum-scale-csi-driver .travis.yml and combines the stages with IBM/ibm-spectrum-scale-csi-opereator's .travis.yml.

- combined stages at job level, so we keep build times down
- renamed Dockerfile.msb as Dockerfile, and removed the old development specific one.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,6 +7,41 @@ go:
...
+    - IMAGE_LABEL=$(
+        if [[ $TRAVIS_EVENT_TYPE == 'cron' ]];
+        then
+            echo "--label version=${IMAGE_VERSION} --label quay.expires-after=2w";
+        else
+            echo "--label version=${IMAGE_VERSION}";
+        fi
+      )
+    - BUILD_DIR_OPERATOR="${TRAVIS_BUILD_DIR}/operator"
+    - BUILD_DIR_DRIVER="${TRAVIS_BUILD_DIR}/driver"
```
Comment (whowutwut) [inline .travis.yml]: 👍

File: .travis.yml [hunk 2]
```diff
@@ -92,39 +137,61 @@ jobs:
...
         - pip install sphinx sphinx_rtd_theme recommonmark 
         - cd ${TRAVIS_BUILD_DIR}/docs
         - make html
+    # end stage: build
 
 
     # Install minikube, build the image and run the scorecard.
     - stage: test
-      name: "Test: scorecard"
+      name: "Test - Opeartor: scorecard"
```
Comment (whowutwut) [inline .travis.yml]: minor spelling issue here: Operator

File: driver/Dockerfile [hunk 3]
```diff
@@ -1,3 +1,12 @@
+# MULTI-STAGE BUILD for IBM Spectrum Scale CSI Driver
+
+FROM golang:1.13.1 AS builder
+WORKDIR /go/src/github.com/IBM/ibm-spectrum-scale-csi-driver/
```
Comment (whowutwut) [inline driver/Dockerfile]: Is this still going against the old repo? or did you clone down the combined repo and give it a different name?
`git clone https://github.com/IBM/ibm-spectrum-scale-csi ibm-spectrum-scale-csi-driver`

File: driver/Dockerfile [hunk 4]
```diff
@@ -8,7 +17,5 @@ LABEL name="IBM Spectrum Scale CSI driver" \
       description="CSI Plugin for IBM Spectrum Scale"\
       maintainers="IBM Spectrum Scale"
 COPY licenses /licenses
-
-COPY _output/ibm-spectrum-scale-csi /ibm-spectrum-scale-csi
-RUN chmod +x /ibm-spectrum-scale-csi
+COPY --from=builder /go/src/github.com/IBM/ibm-spectrum-scale-csi-driver/_output/ibm-spectrum-scale-csi /ibm-spectrum-scale-csi
```
Comment (whowutwut) [inline driver/Dockerfile]: If the above is comment is wrong at line 4, we should update this path too...

Comment (dunnevan) [reply driver/Dockerfile]: Yeah, we should.  I did that in my multi-arch build, but forgot to do it in this version.  Thanks for the catch!

File: driver/Dockerfile (hunk 3 above)
Comment (dunnevan) [reply driver/Dockerfile]: This is just what we name ourselves inside the docker, shouldn't have any impact otherwise.  I will update though

**Pull request merged**